bots/webcomic.bot.js

```diff
@@ -35,22 +35,23 @@ const getComics = (arr, count=3)=>{
 Slack.onMessage(async (msg)=>{
 	try{
 		if(msg.text.startsWith('comic:')){
-			const link = msg.text.split('comic:')[1].trim().replace('<', '').replace('>', '');
-
-			await Gist.append(GistId, {
-				webcomics : [{
-					date: (new Date()).toISOString(),
-					link,
-					user : msg.user,
-					shared : false
-				}]
-			})
-			Slack.send(msg.channel, 'Webcomic saved!');
+			const match = msg.text.match(/<(.+)\|.*>/);
```

Comment:
Also, even though it's just a one line regex, could we extract it into a util? If we want to parse URLs somewhere else, it would be nice to not have to copy this regex around (or worse, forget about this bot and recreate the regex from scratch).